fib.py

```diff
@@ -0,0 +1,21 @@
+def fibonacci(n):
```

Comment:
Consider adding type hints to the function signature. This can improve readability and make the code easier to understand. [medium]